package.json

```diff
@@ -9,6 +9,7 @@
     "fennec": "web-ext run --target=firefox-android --source-dir=dist/firefox-mobile --firefox-apk=org.mozilla.firefox",
     "fennec-nightly": "web-ext run --target=firefox-android --source-dir=dist/firefox-mobile --firefox-apk=org.mozilla.fennec_aurora",
     "package:firefox": "web-ext build --source-dir=dist/firefox --artifacts-dir=bin",
+    "package:chrome": "crx pack dist/chrome -o bin/webcompat-reporter-chrome.crx",
```

Comment:
One thing I just ran into by accident: if you try to package before having run build, you get a confusing error.

Maybe we can call `build:chrome && crx....` to ensure that build always runs?